sections/dashboard/MobileDashboard/OpenPositions.tsx

```diff
@@ -50,15 +50,15 @@ const OpenPositions: React.FC<OpenPositionsProps> = ({
 
 	const POSITIONS_TABS = useMemo(
 		() => [
-			{
-				name: PositionsTab.CROSS_MARGIN,
-				label: t('dashboard.overview.positions-tabs.cross-margin'),
-				badge: crossPositions.length,
-				active: activePositionsTab === PositionsTab.CROSS_MARGIN,
-				detail: formatDollars(portfolio.crossMarginFutures),
-				disabled: false,
-				onClick: () => setActivePositionsTab(PositionsTab.CROSS_MARGIN),
-			},
+			// {
```

Comment:
Be good if we can make cross margin things conditional based on CROSS_MARGIN_ENABLED flag so we don't have to worry about what to reactivate when we eventually get it back in, also leaves the codebase clear of commented code.

Reply:
Good idea, but one place I think we can't do this is the portfolio value on the dashboard. This value will include cross margin free margin for the old contracts unless we comment it out.

Reply:
Can't we just conditionally add free margin to isolated margin in `selectFuturesPortfolio` ?